packages/core/networks/5.json

```diff
@@ -46,5 +46,13 @@
   {
     "contractName": "AddressWhitelist",
     "address": "0xb06C61fcc074Efb8cf7Fd707bC3bd8ab1Aa5A4E9"
+  },
```

Comment:
unrelated to your PR but the `0x20570e9e27920ac5e2601e0bef7244deff7f0b28` `MockOracleAncillary` is listed twice and one can be removed.